src/libs/ReportUtils.ts

```diff
@@ -6356,6 +6356,14 @@ function hasViolations(reportID: string, transactionViolations: OnyxCollection<T
     return transactions.some((transaction) => TransactionUtils.hasViolation(transaction.transactionID, transactionViolations, shouldShowInReview));
 }
 
+/**
+ * Checks to see if a report contains non-hold violations
+ */
+function hasNonHoldViolations(reportID: string, transactionViolations: OnyxCollection<TransactionViolation[]>, shouldShowInReview?: boolean): boolean {
```

Comment:
NAB: technically you just care about one violation
```suggestion
function hasNonHoldViolation(reportID: string, transactionViolations: OnyxCollection<TransactionViolation[]>, shouldShowInReview?: boolean): boolean {
```